File: tests/fixtures/test-project/sample.js
/**
 * Sample JavaScript file for testing
 */

function calculateSum(a, b) {
    /**
     * Calculate the sum of two numbers.
     * @param {number} a - First number
     * @param {number} b - Second number
     * @returns {number} Sum of a and b
     */
    return a + b;
}

function processData(data) {
    return data.toUpperCase();
}

class DataProcessor {
    /**
     * A class for processing data.
     */
    constructor() {
        this.data = [];
    }
    
    /**
     * Add an item to the data list.
     * @param {any} item - Item to add
     */
    addItem(item) {
        this.data.push(item);
    }
    
    /**
     * Get the count of items.
     * @returns {number} Number of items
     */
    getCount() {
        return this.data.length;
    }
}

File: tests/fixtures/test-project/sample.py
#!/usr/bin/env python3
"""
Sample Python file for testing
"""

def calculate_sum(a, b):
    """Calculate the sum of two numbers.
    
    Args:
        a (int): First number
        b (int): Second number
    
    Returns:
        int: Sum of a and b
    """
    return a + b

def process_data(data):
    return data.upper()

class DataProcessor:
    """A class for processing data."""
    
    def __init__(self):
        self.data = []
    
    def add_item(self, item):
        """Add an item to the data list.
        
        Args:
            item: Item to add
        """
        self.data.append(item)
    
    def get_count(self):
        """Get the count of items.
        
        Returns:
            int: Number of items
        """
        return len(self.data)
